app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -184,10 +205,50 @@ public void setConsentNotificationEmail(String consentNotificationEmail) {
     public void setUserProfileAttributes(Set<String> profileAttributes) {
         this.profileAttributes = profileAttributes;
     }
-    
-    @DynamoDBAttribute
+    /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = PasswordPolicyMarshaller.class)
+    @Override
+    public PasswordPolicy getPasswordPolicy() {
+        return passwordPolicy;
+    }
+    @Override
+    public void setPasswordPolicy(PasswordPolicy passwordPolicy) {
+        this.passwordPolicy = passwordPolicy;
+    }
+    /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = EmailTemplateMarshaller.class)
+    @Override
+    public EmailTemplate getVerifyEmailTemplate() {
+        return verifyEmailTemplate;
+    }
+    @Override
+    public void setVerifyEmailTemplate(EmailTemplate template) {
+        this.verifyEmailTemplate = template;
+    }
+    /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = EmailTemplateMarshaller.class)
+    @Override
+    public EmailTemplate getResetPasswordTemplate() {
+        return resetPasswordTemplate;
+    }
+    @Override
+    public void setResetPasswordTemplate(EmailTemplate template) {
+        this.resetPasswordTemplate = template;
+    }
+    /** {@inheritDoc} */
+    @Override
+    public boolean isActive() {
+        return active;
+    }
+    @Override
+    public void setActive(boolean active) {
+        this.active = active;
+    }
+    // Left for legacy support of earlier versions of studies. Will be removed after migration.
     @JsonIgnore
     public String getData() {
```

Comment:
There's a test that takes the getData string, deserializes it back to a json node, and verifies it has all the same values as the study deserialized using the normal fields. So I believe this is verified to be working.